article/templates/article/objects/blog_column.html

```diff
@@ -8,7 +10,7 @@
 {% load wagtailcore_tags %}
 {% load wagtailimages_tags %}
 {% load video_filters %}
-<article class="o-article o-article--column{% if padded %} o-article--padded{% endif %}">
+<article class="o-article o-article--column{% if padded %} o-article--padded{% endif %}" style="background-color: {{article|get_colour}}">
```

Comment:
Is there meant to be a CSS style in the article tag? Can the style be placed in a class, or is it too much?